scripts/flexible-server/auto-shutdown.sh

```diff
@@ -30,7 +30,9 @@ do
           else
   		app_env=PTL		  
 	  fi
-
+	if [[ $app_env == "PTL" ]] || [[ $app_env == "PROD" ]] ; then
```

Comment:
we shouldn't need to add this it just shouldn't be tagged in the other environments.

and will prod actually be in capitals?

Reply:
after pushing the tag across, i have noticed that tag got pushed to **prod** env as well.  Still investitating the root cause.

Reply:
can you fix that before this PR then please